lib/timeline/component/LineGraph.js

```diff
@@ -487,6 +487,7 @@ LineGraph.prototype._updateAllGroupData = function (ids, groupIds) {
       //Copy data (because of unmodifiable DataView input.
       var extended = util.bridgeObject(item);
       extended.x = util.convert(item.x, 'Date');
+      extended.x2 = util.convert(item.x2, 'Date');
```

Comment:
I think `x2` is better called `end`. 

We also need documentation for this new parameter `end`. Can you please extend the docs. Thanks!

In addition to that we can think about allowing also `start` as `x` and deprecate `x` in a later version. This would be more compatible with the timeline.